Source/ConversationV1/Tests/ConversationTests.swift

```diff
@@ -0,0 +1,215 @@
+    var service:     Conversation!
+    var workspaceID: Conversation.WorkspaceID! = "car_demo_1"
+
+    // the initial node and response from Watson
+    let initialMessage = "Hello"
+    let initialResponse = "Welcome to the Watson Car Demo..."
+
+    let secondMessage = "Turn on wipers"
+    let secondResponse = "Your wipers are now on"
+    
+    // invalid parameters for negative tests
+    let invalidWorkspaceID = "9354b734-d5b2-4fd3-bee0-e38adbcab575"
+
+    // timeout for asynchronous completion handlers
+    private let timeout: NSTimeInterval = 30.0
+
+    // MARK: - Conversation Tests
+
+    /// Test the Conversation service by executing each operation with valid parameters.
+    func testConversation() {
+
+        // Send a test message with no history/context
+        sendInitialMessageNoContext()
+
+        // Send a test message with no history/context, and then send a follow-up message
+        sendMessageWithContext()
+        
+        // verify error conditions
+        executeNegativeTests()
+    }
+
+    /// Test the Conversation service by executing each operation with invalid parameters.
+    func executeNegativeTests() {
+
+    }
+
+    // MARK: - Helper functions
+
+    /// Load credentials and instantiate Dialog service
+    override func setUp() {
+        super.setUp()
+
+        // identify credentials file
+        let bundle = NSBundle(forClass: self.dynamicType)
+        guard let url = bundle.pathForResource("Credentials", ofType: "plist") else {
+            XCTFail("Unable to locate credentials file.")
+            return
+        }
+
+        // load credentials from file
+        let dict = NSDictionary(contentsOfFile: url)
+        guard let credentials = dict as? Dictionary<String, String> else {
+            XCTFail("Unable to read credentials file.")
+            return
+        }
+
+        // read Dialog username
+        guard let username = credentials["ConversationUsername"] else {
+            XCTFail("Unable to read Conversation username.")
+            return
+        }
+
+        // read Dialog password
+        guard let password = credentials["ConversationPassword"] else {
+            XCTFail("Unable to read Conversation password.")
+            return
+        }
+
+        // instantiate the service
+        if service == nil {
+            service = Conversation(username: username, password: password)
+        }
+    }
+
+    /// Wait for an expectation to be fulfilled.
+    func waitForExpectation() {
+        waitForExpectationsWithTimeout(timeout) { error in
+            XCTAssertNil(error, "Timeout.")
+        }
+    }
+
+    // MARK: - Positive tests
+
+    /// Create the Dialog application
+    func sendInitialMessageNoContext() {
+        let description = "Sending first message with no context."
+        let expectation = expectationWithDescription(description)
+
+        sendInitialMessage() {
+            (response) in
+                expectation.fulfill()
```

Comment:
We should really be verifying at least some of the content that comes back, especially since so many properties are optional for this service. For example, `XCTAssertEqual(response.output.text, "Hello, World!")`.

If you print out `response`, do you see any properties?